recipes/pipewire/all/conanfile.py

```diff
@@ -0,0 +1,431 @@
+        tc.project_options["selinux"] = feature("with_selinux")
+        tc.project_options["session-managers"] = []
+        tc.project_options["sndfile"] = feature("with_libsndfile")
+        tc.project_options["spa-plugins"] = "enabled"
+        tc.project_options["sysconfdir"] = "res"
+        tc.project_options["systemd"] = "disabled"
+        tc.project_options["systemd-user-service"] = "disabled"
+        tc.project_options["tests"] = "disabled"
+        tc.project_options["udev"] = feature("with_libudev", True)
+        tc.project_options["udevrulesdir"] = os.path.join(
+            "res", "udev", "rules.d"
+        )
+        tc.project_options["vulkan"] = feature("with_vulkan")
+        tc.project_options["x11"] = feature("with_x11")
+        tc.project_options["x11-xfixes"] = feature("with_xfixes")
+        tc.generate()
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+
+    def build(self):
+        self._patch_sources()
+        meson = Meson(self)
+        meson.configure()
+        meson.build()
+
+    @property
+    def _libpipewire_api_version_txt(self):
+        return os.path.join(self.package_folder, "res", "libpipewire-api-version.txt")
+
+    @property
+    def _libspa_api_version_txt(self):
+        return os.path.join(self.package_folder, "res", "libspa-api-version.txt")
+
+    def package(self):
+        copy(
+            self,
+            "COPYING",
+            self.source_folder,
+            os.path.join(self.package_folder, "licenses"),
+        )
+        meson = Meson(self)
+        meson.install()
+
+        pkconfig_dir = self.package_path.joinpath("lib", "pkgconfig")
+        libpipewire_pc_file = next(pkconfig_dir.glob("libpipewire-*.pc"))
+        libspa_pc_file = next(pkconfig_dir.glob("libspa-*.pc"))
+
+        libpipewire_api_version = libpipewire_pc_file.stem.split("-")[1]
+        libspa_api_version = libspa_pc_file.stem.split("-")[1]
+
+        save(self, self._libpipewire_api_version_txt, libpipewire_api_version)
+        save(self, self._libspa_api_version_txt, libspa_api_version)
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "lib"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+
+        fix_apple_shared_install_name(self)
+
+    def package_info(self):
+        libpipewire_api_version = load(self, self._libpipewire_api_version_txt).strip()
+        libspa_api_version = load(self, self._libspa_api_version_txt).strip()
+
+        self.runenv_info.define(
+            "PIPEWIRE_CONFIG_DIR", os.path.join(self.package_folder, "res", "pipewire")
+        )
+        self.runenv_info.define(
+            "PIPEWIRE_MODULE_DIR",
+            os.path.join(
+                self.package_folder, "lib", f"pipewire-{libpipewire_api_version}"
+            ),
+        )
```

Comment:
Absolute nitpicking, but could you set the column limit to 100 or 120 for `black`, please?
The line wrapping is a bit excessive and only hurts readability, imo.

Reply:
I reformatted with a longer line length with Ruff. It would be great to have formatting configured for the repository so no one has to think about this kind of thing.

Reply:
If your IDE supports .editorconfig then it is configured https://github.com/conan-io/conan-center-index/blob/2c3904ff13e23e1358e36197e97c90c2188138da/.editorconfig#L12